src/pages/index.js

```diff
@@ -84,24 +90,148 @@ function handleImageClick(data) {
 function handleProfileEditSubmit(formValues) {
   userInfo.setUserInfo({
```

Comment:
You need to update `DOM` only in `then` blocks after a successful response from the server so that when there is an error, it shouldn’t change